.github/workflows/release.yml

```diff
@@ -1,66 +1,81 @@
 name: release
 
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
+  # Only run this job if the "build" job passes
+  workflow_run:
+    workflows: [build]
+    types: [completed]
 
 jobs:
-  release:
-    # Only run this job if the "build" job passes
-    needs: build
-
+  on-success:
     if: github.event_name == 'push' && github.ref == 'refs/heads/main'
-    
-  # Set up operating system
-    runs-on: ${{ matrix.os }}
```

Comment:
I think it's fine to keep this part in - this works on CI so should also probably work on the other 2 os

Reply:
Oh Tiffany said we suggest not to put other OS for now and that we usually build it for ubuntu so skipped it for now.

Reply:
Cooool all good!